stomp_core/include/stomp_core/utils.h

```diff
@@ -104,18 +105,18 @@ static const int FINITE_DIFF_RULE_LENGTH = 7;
 
 /** @brief Contains the coefficients for each of the finite central differentiation (position, velocity, acceleration, and jerk) */
 static const double FINITE_CENTRAL_DIFF_COEFFS[FINITE_DIFF_RULE_LENGTH][FINITE_DIFF_RULE_LENGTH] = {
-    {0, 0        , 0        , 1        , 0        , 0         , 0      }, // position
-    {0, 1.0/12.0 , -2.0/3.0 , 0        , 2.0/3.0  , -1.0/12.0 , 0      }, // velocity
-    {0, -1/12.0  , 16/12.0  , -30/12.0 , 16/12.0  , -1/12.0   , 0      }, // acceleration (five point stencil)
-    {0, 1/12.0   , -17/12.0 , 46/12.0  , -46/12.0 , 17/12.0   , -1/12.0}  // jerk
+    {0    , 0        , 0        , 1        , 0        , 0         , 0      }, // position
+    {0    , 1/12.0   , -2/3.0   , 0        , 2/3.0    , -1/12.0   , 0      }, // velocity
+    {0    , -1/12.0  , 16/12.0  , -30/12.0 , 16/12.0  , -1/12.0   , 0      }, // acceleration (five point stencil)
+    {1/8.0, -1       , 13/8.0   , 0        , -13/8.0  , 1         , -1/8.0 }  // jerk
```

Comment:
Wow that last row was so off! glad you caught that

Reply:
I really welcome these changes,  however OpenMP's coding rigidity seems to make it impossible to track a class member from within the parallelized methods, this means that it won't be possible to stop the planner while it's planning.  Anyone knows any workarounds for this?

Reply:
This should still be applied, regardless of OpenMP right?

Reply:
yes